changelogs/fragments/58973_luks_device-add-label-and-uuid-support.yml

```
@@ -0,0 +1,2 @@
+minor_changes:
+- luks_device - added support to use uuid and labels with LUKS2 containers
```

```suggestion
- luks_device - added support to use UUIDs, and labels with LUKS2 containers
```
UUIDs also work with LUKS1 containers

---

Done, thanks!